modules/gumgumBidAdapter.js

```diff
@@ -85,6 +85,7 @@ function _getBrowserParams(topWindowUrl) {
     sw: topScreen.width,
     sh: topScreen.height,
     pu: stripGGParams(topUrl),
+    wl: window.location.href || '',
```

Comment:
What's wrong with other fields available in https://github.com/prebid/Prebid.js/blob/c0d565845dd8749b794807287cc28f5c5d902644/src/refererDetection.js#L238

Reply:
This anomaly has only been noticed with the GG adapter with one of our pubs, we receive localhost as the page URL in some cases (can't provide in which exact cases) when we are getting pu = bidderRequest.refererInfo.page. This way we can also validate that window.location.href matches with what we get from refererInfo.

Reply:
but you get that from this right? https://github.com/prebid/Prebid.js/blob/c0d565845dd8749b794807287cc28f5c5d902644/src/refererDetection.js#L243